//每日签到
package signIn;
option py_generic_services=true;
import "universal/public.proto";
import "common.proto";

message signInType{
	required uint32 signType=1;//签到类型（1：每日签到，2：帮派签到）
}

message signInMsg{
	required uint32 signCount=1;//签到次数
	required uint32 signIn=2;//今天是否已签到 （0：未签到 1：已签到）
	required uint32 signAgain=3;//今天是否有补签 （0：否 1：是）
	optional uint32 signType=4;//签到类型（1：每日签到，2：帮派签到）
}


//上行(客户端主动发起)
service terminal2main{
  	rpc rpcSignInOpen(signIn.signInType)returns(public.fake);//请求打开签到界面
  	rpc rpcSignIn(signIn.signInType)returns(public.fake);		//签到
}

//下行(服务端主动发起)
service main2terminal{
	rpc rpcSignInMsg(signIn.signInMsg)returns(public.fake);//签到次数
}